onnx/backend/test/case/node/string_concat.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import onnx
+from onnx.backend.test.case.base import Base
+from onnx.backend.test.case.node import expect
+
+
+class StringConcat(Base):
+    @staticmethod
+    def export() -> None:
```

Comment:
Test empty strings too?

empty + empty
empty + something
something + empty

Reply:
Also non-ascii characters

Reply:
Added tests for both